pkg/connector/tickets.go

```diff
@@ -542,3 +518,36 @@ func (d *Connector) generateIssueURL(issueKey string) (string, error) {
 	baseURL.Path = path.Join("browse", issueKey)
 	return baseURL.String(), nil
 }
+
+// TODO(lauren) add this to go-jira fork
+func findIssueTypeFromProject(project *jira.Project, issueTypeId string) *jira.IssueType {
+	for _, issueType := range project.IssueTypes {
+		if issueType.ID == issueTypeId {
+			return &issueType
+		}
+	}
+	return nil
+}
+
+func GetProjectAnnotation(annotations []*anypb.Any) *pt.IssueTypeProject {
+	pta := &pt.IssueTypeProject{}
+	for _, a := range annotations {
+		if a.MessageIs(pta) {
+			err := a.UnmarshalTo(pta)
+			if err != nil {
+				return nil
+			}
+			return pta
+		}
+	}
+	return nil
+}
+
+func GetProjectKeyAndIssueTypeIDFromSchemaID(schemaID string) (string, string, error) {
+	// Format is projectKey:issueID
+	schemaIDParts := strings.Split(schemaID, ":")
```

Comment:
Yes I think this should maybe be a type instead of a function and then it can be handled always by it. Just a thought